backend/tasks/populate-date-ranges/populate-blank-date-ranges.js

```diff
@@ -173,11 +173,11 @@ export async function populateBlankDateRangesForYear(
       `${dateRangeKeys.length} applicable DateRanges. Checking if any exist already...`,
     );
 
-    // Exclude the Operating date type,
+    // Exclude the Park gate open date type,
     // handled by `populateBlankGateOperatingDates` in ./populate-blank-gate-dates.js
-    const dateTypeIdWhere = parkDateTypesByName.Operating
+    const dateTypeIdWhere = parkDateTypesByName["Park gate open"]
```

Comment:
is there any way to use the `strapiDateTypeId` here instead of the name? This will be very fragile if someone changes the name.